### PR TITLE
Add \plim

### PR DESCRIPTION
**Pull request by elbaro**
Add \plim from statmath package.

plim

Also moved `\argmin` in the documentation to the right column as it can take `\limits`.

### CODE REVIEW

File: docs/supported.md [hunk 1]
```diff
@@ -361,7 +361,8 @@ Direct Input: $+ - / * ⋅ ± × ÷ ∓ ∔ ∧ ∨ ∩ ∪ ≀ ⊎ ⊓ ⊔ ⊕
 | $\cosec$ `\cosec`  | $\hom$ `\hom`  | $\tg$ `\tg`  | $\Pr$ `\Pr`  |
 | $\cosh$ `\cosh`| $\ker$ `\ker`  | $\th$ `\th`  | $\sup$ `\sup` |
 | $\cot$ `\cot` | $\lg$ `\lg`| $\operatorname{f}$ `\operatorname{f}` | $\argmax$ `\argmax` |
-|$\argmin$ `\argmin`|
+| | | |$\argmin$ `\argmin`|
+| | | | $\plim$ `\plim` |
```
Comment (kevinbarabash) [inline docs/supported.md]: I think it's fine for `argmin` to appear first.  If you want organize this table better a separate PR would be preferable.

File: test/katex-spec.js [hunk 2]
```diff
@@ -3165,6 +3165,10 @@ describe("A macro expander", function() {
         expect`\liminf`.toParseLike`\mathop{\operatorname{lim\,inf}}\limits`;
     });
 
+    it("should expand \\plim as expected", () => {
+        expect`\plim`.toParseLike`\mathop{\operatorname{plim}}\limits`;
+    });
```
Comment (kevinbarabash) [inline test/katex-spec.js]: Thanks for adding a test.

**Comment by kevinbarabash** [comment]
Thanks for the PR.

**Comment by ylemkimon** [comment]
[clabot:check]

**Pull request merged**